docs/waivers.md

```diff
@@ -26,13 +26,13 @@ Waiver files are [input files](https://www.inspec.io/docs/reference/inputs/) wit
 ```yaml
 control_id:
   expiration_date: YYYY-MM-DD
-  skip: true
+  run: true
```

Comment:
Since this is the waiver grammar, should this be flipped to `false`?

Reply:
I'm going to leave it as true and expand the explanation.